config/crd/bases/logging.openshift.io_clusterloggings.yaml

```diff
@@ -368,6 +368,25 @@ spec:
                             description: TimeUnit is a time unit like h,m,d
                             pattern: ^([0-9]+)([yMwdhHms]{0,1})$
                             type: string
+                          namespaceSpec:
+                            description: The per namespace specification to delete documents older than a given minimum age
+                            items:
+                              properties:
+                                minAge:
+                                  description: Delete the records matching the namespaces which are older than this MinAge (e.g. 1d)
+                                  pattern: ^([0-9]+)([yMwdhHms]{0,1})$
+                                  type: string
+                                namespace:
```

Comment:
if this is to be mostly used a prefix, name it as a prefix , like`nsPrefix`
usage should be clear from name. a prefix can be understood to be used as exact name, but exact name may not be interpreted to be used as prefix.